modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -1,20 +1,16 @@
 # ftpsync is a Debian tool for reliably mirroring Debian archives:
 # https://www.debian.org/mirror/ftpmirror
 define ocf_mirrors::ftpsync(
-    $cron_minute,
     $rsync_host,
+    $cron_minute = undef,
```

Comment:
No, if isn't set, then [cron isn't even used at all](https://github.com/ocf/puppet/pull/114/files#diff-8c8f6ab7251650b2233929d55bdb08f1R45), which makes it possible to define a push-only triggered ftpsync for example. Not super useful right now, but I thought we might want to do this sometime with Kali if push mirroring works fine.